src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/managehearings/ManageHearingActionService.java

```diff
@@ -162,19 +164,17 @@ private void addDocumentsToCollection(Map<String, DocumentRecord> documentMap,
      */
     public void updateTabData(FinremCaseData caseData) {
         ManageHearingsWrapper hearingsWrapper = caseData.getManageHearingsWrapper();
-        List<ManageHearingsCollectionItem> hearings = Optional.ofNullable(hearingsWrapper.getHearings())
-            .orElseGet(ArrayList::new);
+        List<ManageHearingsCollectionItem> hearings = hearingsWrapper.getHearings();
 
         List<HearingTabCollectionItem> hearingTabItems = mapAndSortHearings(hearings, caseData);
 
-        Map<String, List<HearingTabCollectionItem>> partyTabItems = Map.of(
-            APPLICANT, filterHearingTabItems(hearingTabItems, APPLICANT),
-            RESPONDENT, filterHearingTabItems(hearingTabItems, RESPONDENT),
-            INTERVENER1, filterHearingTabItems(hearingTabItems, INTERVENER1),
-            INTERVENER2, filterHearingTabItems(hearingTabItems, INTERVENER2),
-            INTERVENER3, filterHearingTabItems(hearingTabItems, INTERVENER3),
-            INTERVENER4, filterHearingTabItems(hearingTabItems, INTERVENER4)
-        );
+        Map<String, List<HearingTabCollectionItem>> partyTabItems = new LinkedHashMap<>();
+        partyTabItems.put(APPLICANT, filterHearingTabItems(hearingTabItems, APPLICANT));
+        partyTabItems.put(RESPONDENT, filterHearingTabItems(hearingTabItems, RESPONDENT));
+        partyTabItems.put(INTERVENER1, filterHearingTabItems(hearingTabItems, INTERVENER1));
+        partyTabItems.put(INTERVENER2, filterHearingTabItems(hearingTabItems, INTERVENER2));
+        partyTabItems.put(INTERVENER3, filterHearingTabItems(hearingTabItems, INTERVENER3));
+        partyTabItems.put(INTERVENER4, filterHearingTabItems(hearingTabItems, INTERVENER4));
```

Comment:
Fine, but will look out for your change to populate this based on what PartiesService say is on the case.